src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -1,14 +1,26 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
+import { isEmpty } from 'lodash';
 import * as React from 'react';
+
+import { Screenshot } from './screenshot';
 import { ScreenshotViewModel } from './screenshot-view-model';
 
-export type ScreenshotViewProps = { viewModel: ScreenshotViewModel };
-export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', props => {
+export type ScreenshotViewProps = {
+    viewModel: ScreenshotViewModel;
+};
+
+const screenshotAltText: string = 'axe-android results screenshot with highlighted components';
+
+export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', (props: ScreenshotViewProps) => {
+    if (isEmpty(props.viewModel.screenshotData)) {
+        return <div>Screenshot for scan is unavailable</div>;
+    }
     return (
         <div>
-            <h1 tabIndex={0}>This is the screenshot view</h1>
+            <h1 tabIndex={0}>Target Page Screenshot</h1>
```

Comment:
We still have the `tabIndex` here... I think my original comment got lost somewhere

Reply:
This is intentional because there has to be an element in tab order inside the independently scrollable region to accomodate https://dequeuniversity.com/rules/axe/3.3/scrollable-region-focusable - it's not sufficient for it to be arrow/header navigable since the rule is still applicable for non-screen-reader keyboard users

Reply:
leaving a comment sounds good.

Reply:
updating and adding the comment to link to the axe-core 3.3. rule on scrollabe regions!